app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file, new_file = command.split()
+    if "mv" in command:
+        directory = new_file.split("/")
+        parent = ""
+        if len(directory) > 1:
+            for i in range(len(directory) - 1):
```

Comment:
no need to use range here, u can make it in such way: "for path in directory[:-1]:" and make os.path.join with parent + path